app/assets/stylesheets/resources.css.scss

```diff
@@ -0,0 +1,89 @@
+.resources {
+  list-style-type: none;
+}
+
+h3 {
+  margin: 10px 0 10px 5px;
+}
+  
+.resource-cards {
+  @include span-columns(6);
+  @include omega(2n);
+  background-color: $light-tan;
+  box-shadow: 10px 10px 30px 0px rgba(0,0,0,0.75);
+  height: 250px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+  position: relative;
+
+  p {
+    margin: 2px 0 2px 0;
+  }
+
+  .map {
+    @include span-columns(6 of 6);
+    bottom: 0;
+    position: absolute;
+  }
+
+  .vet-details-left {
+    @include span-columns(3 of 6);
+    @include pad();
+    padding-top: 0px;
+    text-align: left;
+  }
+
+  .vet-details-right {
+    @include span-columns(3 of 6);
+    @include reset-display;
+    @include pad();
+    @include omega();
+    margin-top: 25px;
+  }
+}
+
+.resource-title {
+  @include span-columns(12);
+}
+
+.resource-cards-small {
+  @include span-columns(4);
+  @include omega(3n);
+  background-color: $light-tan;
+  box-shadow: 10px 10px 30px 0px rgba(0,0,0,0.75);
+  margin-bottom: 10px;
+  margin-top: 10px;
+  min-height: 100px;
+
+  p {
+    margin-bottom: 2px;
+    padding-left: 10px;
+  }
+}
+
+aside {
+  @include span-columns(12);
+  @include shift(0);
+  $base-spacing: 1.5em !default;
+  margin: 50px 0 40px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `50px 0 40px` instead of `50px 0 40px 0`